coral/src/app/features/approvals/acls/AclApprovals.tsx

```diff
@@ -379,12 +388,9 @@ function AclApprovals() {
             setRejectModal({ isOpen: true, reqNo: detailsModal.reqNo });
           }}
           isLoading={approveIsLoading}
+          disabledActions={selectedRequest?.requestStatus !== "CREATED"}
```

Comment:
Same here, I would add a test case 🥺

Reply:
This is the test case I think? :o https://github.com/aiven/klaw/pull/666#discussion_r1114055743

Reply:
It wasn't the case, this is the case [9639660](https://github.com/aiven/klaw/pull/666/commits/96396608c953620e1c368fe33b356e6ddf15c98f)